javascript/react-js/react-introduction.md

```diff
@@ -239,3 +244,4 @@ This section contains questions for you to check your understanding of this less
 - <a class="knowledge-check-link" href="https://dev.to/sarah_chima/an-introduction-to-react-components-cke">How should you structure your application into components?</a>
 - <a class="knowledge-check-link" href="#create-react-app">What does `create-react-app` do?</a> 
 - <a class="knowledge-check-link" href="#testing">What is commonly used for testing React apps?</a> 
+- <a class="knowledge-check-link" href="#react-developer-tools">What can you do with React Developer Tools?</a>
```

Comment:
I think this could be improved, as in its current state this question is very broad/vague.

One suggestion off the top of my head would be to ask something that compares it to the information in the regular dev tools.